src/drivers/ulanding/ulanding.cpp

```diff
@@ -83,7 +136,11 @@ class Radar : public device::CDev
 	void task_main();
 
 	bool read_and_parse(uint8_t *buf, int len, float *range);
+#if defined(__PX4_POSIX)
+	bool is_header_byte(uint8_t c) {return (c == ulanding_hdr);};
```

Comment:
@dagar This wasn't necessary for posix but for the new version of the ulanding sensor. I will change some things to make it more streamlined and obvious.